src/armscan_env/volumes/loading.py

```diff
@@ -0,0 +1,17 @@
+import SimpleITK as sitk
+
+
+def load_sitk_volume(
```

Comment:
is this now being used everywhere for loading images?

Reply:
not yet